src/App.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Routes,
+} from 'react-router-dom';
+import { AuthProvider, RequireAuth } from 'react-auth-kit';
+import Login from './Components/auth/SignInForm';
+import Register from './Components/auth/SignUpForm';
+import AddGamesForm from './Components/games/AddGamesForm';
+import Session from './Components/session/Session';
+import MainPage from './Components/games/MainPage';
+import DeleteGame from './Components/games/DeleteGame';
+import Reservations from './Components/reservations/Reservations';
+import GameDetails from './Components/details/GameDetails';
+import './App.css';
+import GamesListComponent from './Components/games/GamesListComponent';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import ParentComponent from './Components/reservations/ReservationForm';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main App functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the rest of the functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/cancelei/EsportGamingCenter_frontend/assets/57421823/4f0451d2-cae7-4616-96c9-7cf83356b57e)


That way, your code will meet the latest standards. 💪

Reply:
Now all the functions have been refactored. 👨🏼‍🚀
![image](https://github.com/cancelei/EsportGamingCenter_frontend/assets/45726631/c4f8f053-3c80-49f1-b352-baeb6d184431)